addons/addon-base-raas/packages/base-raas-services/lib/data-egress/data-egress-service.js

```diff
@@ -59,6 +68,7 @@ class DataEgressService extends Service {
 
   async createEgressStore(requestContext, environment) {
```

Comment:
btw, I am assuming that this won't be an API that is exposed by SWB.

Reply:
Yes, you are right, this would only be served as an internal method called during environment provisioning.